scapy/utils.py

```diff
@@ -2043,13 +2074,21 @@ def _get_time(self,
 
         return sec, usec  # type: ignore
 
-    def build_block(self, block_type, block_body):
-        # type: (bytes, bytes) -> bytes
+    def _add_padding(self, raw_data):
+        # type: (bytes) -> bytes
+        if len(raw_data) % 4 != 0:
+            padding = b"\x00" * (4 - len(raw_data) % 4)
+            raw_data += padding
```

Comment:
```suggestion
        raw_data += ((-len(raw_data)) % 4) * b"\x00"
```
Has the minimum amount of operations